src/components/toast/toast.component.js

```diff
@@ -128,6 +134,10 @@ const Toast = React.forwardRef(
             {...tagComponent(restProps["data-component"] || "toast", restProps)}
             {...toastProps}
             ref={toastContentNodeRef}
+            tabIndex={-1}
+            role="status"
```

Comment:
I might be missing something here but why does this need to be a live region? The content should already be announced by a screenreader when it appears, due to the auto focus.

Reply:
I added this after reading this article on Toast best practices https://adrianroselli.com/2020/01/defining-toast-messages.html#NeededFeatures
`alert` and `log` don't fit the uses of toast as `alert` will interrupt anything the user is doing and read out in an assertive style. I can probably remove the `aria-atomic` as that is implicit with `status`.

Reply:
I still say remove the `role="status"` as well as the `aria-atomic` (which only applies to live regions anyway) - I think you've misunderstood that part of the article. It's talking about the *lots* of different ways of implementing what are vaguely known as "toast" notifications. Some simply pop some text up at the top/bottom of the screen, and in that case a live region is probably the best way. We're using what is effectively a dialog, which is given focus when it appears, so screenreaders already pick it up. In addition, the article underneath the part you linked to says `Generally live regions must exist on the page at render`, which means they won't even work with a dialog that only pops into existence at the point the toast message appears.

Unless I'm missing something, and you experienced specific issues when testing with a screenreader which this solved, I say get rid of it.